File: Effective OC 52/Effective OC 52/ViewController.m
//
//  ViewController.m
//  Effective OC 52
//
//  Created by kfz on 2017/3/21.
//  Copyright © 2017年 kongfz. All rights reserved.
//

#import "ViewController.h"
#import "RTCar.h"
#import "RTCar+getSet.h"

@interface ViewController ()

@end

@implementation ViewController



- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    [RTCar logIvars];
    [RTCar logMethodList];
    
    
}



@end

File: swift-2048/swift-2048/ViewController.swift
//
//  ViewController.swift
//  swift-2048
//
//  Created by kfz on 16/9/15.
//  Copyright © 2016年 kognfz. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    
    @IBAction func starGAME(sender: UIButton) {
        let alert = UIAlertController(title: "开始", message: "准备好了吗？", preferredStyle: UIAlertControllerStyle.Alert)
        alert.addAction(UIAlertAction(title: "开始游戏 ", style: UIAlertActionStyle.Default, handler: {
            action in
            self .presentViewController(JLTabBarViewController(), animated: true, completion: nil)
        }))
        self.presentViewController(alert, animated: true, completion: nil)
    }

    

}

